### ISSUE TITLE
Error while processing 

### ISSUE THREAD

**Issue by DaBaiTuu** [DLYuanGod/TinyGPT-V#21]
**Describe the bug and Screenshots**
聊天界面 error，有问题, 终端有警告信息,
`trainable params: 31457280 || all params: 2811233280 || trainable%: 1.1189850455953623
Loading Q-Former
Loading Q-Former Done
768 2560
Load MiniGPT-4 Checkpoint: /home/dongsongb/PYTHON-PROJ/openai/TinyGPT-V/TinyGPT-V_for_Stage3.pth
Initialization Finished
/home/dongsongb/PYTHON-PROJ/openai/TinyGPT-V/demo.py:173: GradioDeprecationWarning: The `enable_queue` parameter has been deprecated. Please use the `.queue()` method instead.
  demo.launch(share=True, enable_queue=True)
Running on local URL:  http://127.0.0.1:7860
/home/dongsongb/anaconda3/envs/AutoML/lib/python3.10/site-packages/transformers/generation/utils.py:1518: UserWarning: You have modified the pretrained model configuration to control generation. This is a deprecated strategy to control generation and will be removed soon, in a future version. Please use and modify the model generation configuration (see https://huggingface.co/docs/transformers/generation_strategies#default-text-generation-configuration )
  warnings.warn(
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.`
如图所示
![Screenshot from 2024-01-10 13-52-30](https://github.com/DLYuanGod/TinyGPT-V/assets/120782756/472a20dc-8707-4657-b4e1-2ef76e69156e)
![Screenshot from 2024-01-10 13-52-18](https://github.com/DLYuanGod/TinyGPT-V/assets/120782756/5909279f-5c03-481d-96ff-1b4bb77b6a42)

**To Reproduce**
1. just run domo.py
2. Click on 'link' to open
3. upload image and input.

**Screenshots**

**Desktop:
 - OS: Ubuntu18.04
 - Browser firefox

**Comment by llizhaoxu** [comment]
Hi

Did you use our environment.yml to bulid the env? Please check your Gradio version. Thanks for your interest!

**Comment by DaBaiTuu** [comment]
> Hi
> 
> Did you use our environment.yml to bulid the env? Please check your Gradio version. Thanks for your interest!

sorry ，确实不是直接从 environment.yml 创建的，是补充安装的库， 然后我单独升级了 gradio 之后还是有些问题， 
再然后我直接 重建了环境确实没有问题了，的确不能偷懒阿，谢谢
